### PR TITLE
Update Gems

### PR DESCRIPTION
**Pull request by michaelwehrley**
* Update gems to rubygem repos
* Update host app gems (`jasmine-rails` & `brakeman`) to match repos

[#110176596]

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -33,7 +33,7 @@ gem "bit_player",
     tag: "0.4.9",
     git: "https://github.com/cbitstech/bit_player.git"
 gem "social_networking",
-    tag: "0.9.1",
+    tag: "0.9.2",
```
Comment (michaelwehrley) [inline Gemfile]: @ericcf I upload `social_networking` 0.9.2 to rubygems.  However, it seemed to fail locally and on travis.  Yet, when I referenced the git 0.9.2 version, it passed.  So I `yanked` 0.9.2 from rubygems.  Not sure what the problem was.

Comment (ericcf) [inline Gemfile]: What commands did you use to publish it? Where was the exception being thrown?

Comment (ericcf) [inline Gemfile]: You should be able to create and reference the gem locally for testing as well before publishing to rubygems.

Comment (michaelwehrley) [inline Gemfile]: I used the steps I mentioned here: https://github.com/cbitstech/social_networking#publishing-to-rubygems.  I have tested it locally and all tests pass in the engine and in the host app locally.  I will try again when I tag it again.

**Comment by ericcf** [comment]
Do you want this merged for now?

**Comment by michaelwehrley** [comment]
I think that makes sense.  thanks!

**Pull request merged**